### ISSUE TITLE
TAC repo uses the wrong license (Apache vs CC)

### ISSUE THREAD

**Issue by lehors** [ossf/tac#88]
The LICENSE file in the TAC repo currently contains the Apache Software license. Given that the TAC isn't going to develop code though it should contain the Creative Common Attribution license instead. This will be applicable to any documentation the TAC develops.

**Comment by joshbressers** [comment]
I approve of this change mechanically, but to avoid future legal concerns, do we need to get a sign off or approval from someone with legal authority at the Linux Foundation?

I suspect all contributions up to this point are owned by the contributors and covered under Apache 2. I'm unsure how that would affect this move.

@david-a-wheeler @brianbehlendorf perhaps one of you can weigh in

**Comment by david-a-wheeler** [comment]
@joshbressers - fair concern.

I'm not a lawyer. However, there isn't much text under the TAC currently, and the individual contributions are likely de minimus. We could probably just switch. However, if we want to be cautious, we could ask all the current & previous TAC members to approve release under CC-BY-4.0 via email & then we're fully covered for their contributions. I don't think anyone else has contributed directly to TAC stuff.

To avoid future problems, perhaps the LICENSE text should say:

> Unless otherwise noted, all text released directly from the OpenSSF TAC is licensed under the Creative Commons Attribution license (CC-BY-4.0), source code is released under the Apache-2.0 or MIT license, and standards are released under Community Specification License. For the license of a specific OpenSSF project, see that project. See the OpenSSF Charter for OpenSSF license policies.

Then under the file LICENSES, add each license using its SPDX id as its filename. That's the usual way to do this. Community Specification license Version 1.0 is available at https://github.com/CommunitySpecification/1.0 

BTW, the SPDX id for the current Creative Commons Attribution license is CC-BY-4.0 <https://spdx.org/licenses/>.

**Comment by lehors** [comment]
I agree that @joshbressers raises a valid question. We don't need approval to switch license from all tac members though, only those who have contributed content to this repo. I suspect this is a lot fewer. And the answer is in the git log. :-)
I'm away from my computer right now but could look into it.
@david-a-wheeler As for the actual text, I don't think we can have a "this license or that license" kind of statement. It's got to be definitive otherwise one wouldn't know which applies to any content without an explicit embedded license (or SPDX I'd).

**Comment by david-a-wheeler** [comment]
@lehors - no law requires machine-processable text in a LICENSE file. My worry is that if we drop CC-BY into LICENSE, we'll have the same problem if the TAC generates something else.

**Comment by lehors** [comment]
I understand although I'm doubtful anything else will be produced by the TAC. But let me clarify: I was specifically referring to the Apache or MIT piece as being a problem. I think it's fine to provide for different licenses that apply to different types of content, but not two possible licenses applying to the same type.

**Comment by david-a-wheeler** [comment]
@lehors - You're probably right that the TAC probably won't produce anything else, but I'd rather avoid unintentional problems. As far as Apache and MIT goes, how about this instead?

> Unless otherwise noted, all ordinary text released directly from the OpenSSF TAC is licensed under the Creative Commons Attribution license (CC-BY-4.0), source code is released under the Apache-2.0 license, and standards are released under Community Specification License 1.0. For the license of a specific OpenSSF project, see that project. See the OpenSSF Charter for OpenSSF license policies.

It's true that we only have to ask people who previously contributed.

**Comment by lehors** [comment]
Yes, i think that works.

**Comment by lehors** [comment]
So here is the list of the people who've made contributions to this repo:
```
Bob Callaway <bcallaway@google.com>
Chris Aniszczyk <caniszczyk@gmail.com>
Dan Lorenc <lorenc.d@gmail.com>
Dan Middleton <dan.middleton@intel.com>
David A. Wheeler <dwheeler@linuxfoundation.org>
Dirk Mueller <dirk@dmllr.de>
Fatih Degirmenci <fatih.degirmenci@ericsson.com>
Kay Williams <kayw@microsoft.com>
LindsayLF <69853426+LindsayLF@users.noreply.github.com>
Luke Hinds <7058938+lukehinds@users.noreply.github.com>
Marcin Hoppe <marcin.hoppe@auth0.com>
Maya Kaczorowski <15946341+mayakacz@users.noreply.github.com>
Phil Estes <estesp@gmail.com>
Ryan Haning <ryhaning@microsoft.com>
Todd Benzies <tbenzies@linuxfoundation.org>

```
It should be pretty straightforward to get their ok. @david-a-wheeler I imagine you know who LindsayLF and Maya Kaczorowski are?

**Comment by david-a-wheeler** [comment]
LindsaySF is Lindsay Mays Gendreau, the first OpenSSF PM. Her work was for the LF (as an employee), so I think the LF can speak for her contributions. But if you really want to contact her, I think this works: lgendreau [at] linuxfoundation {dot.} org - I'm pretty she's still around here just in a different role.

Maya Kaczorowski has GitHub page <https://github.com/mayakacz>, & was previously at GitHub & Google Cloud Platform. Her web page is here: https://mayakaczorowski.com/ I think I have her email address but I can't give it out publicly. If you need my help contacting her let me know.

**Comment by bobcallaway** [comment]
@jorydotcom can i ask for your help in getting the explicit approvals for the license change from the list of folks above?

Thanks!

**Comment by lehors** [comment]
I'm not sure anyone cares but for what it's worth I submitted PR #143 in an attempt to address this issue without having to chase former contributors.
